### PR TITLE
PR for including Frog nodes in the staking dashboard

### PR DESCRIPTION
**Pull request by frogstaking**
PR for including Frog nodes in the staking dashboard

### CODE REVIEW

File: library/validator-assets/src/Frog.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+const Frog = () => (
+  <svg height="800px" width="800px" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"
```
Comment (rossbulat) [inline library/validator-assets/src/Frog.tsx:2]: I ran SVGO on this SVG, would you be able to replace with this minimised version?:
```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><path d="M14 279.998c0 47.353 76.181 138 242 138s242-90.647 242-138H14z" fill="#bee178"/><path d="M256 283.998H12s0 36 42.069 36c33.655 0 117.793-18 201.931-18s168.276 18 201.931 18c42.069 0 42.069-36 42.069-36H256z" fill="#d2f0a0"/><path d="M492.818 243.208c-8.897-6.575-20.211-14.215-32.84-22.258l-1.183-12.903C467.004 197.438 472 183.449 472 168.109c0-19.917-8.417-37.559-21.365-48.472-16.084-29.208-61.067-36.693-86.915-16.053C328 132.109 294.667 148.109 256 148.109s-72-16-107.72-44.524c-25.848-20.64-70.83-13.155-86.915 16.053C48.417 130.551 40 148.193 40 168.109c0 16.475 5.759 31.396 15.075 42.238l-.622 9.059c-13.608 8.608-25.808 16.807-35.271 23.801C12.855 247.883 8 255.381 8 264.459c0 13 8.782 23.539 19.615 23.539h456.77c10.833 0 19.615-10.539 19.615-23.539 0-9.078-4.855-16.576-11.182-21.251z" fill="#82c35f"/><g fill="#96cd78"><ellipse cx="92" cy="168.11" rx="52" ry="60"/><ellipse cx="420" cy="168.11" rx="52" ry="60"/></g><path d="M472 268.109H40a8 8 0 000 16h432a8 8 0 000-16zM320 300.109H192a8 8 0 000 16h128a8 8 0 000-16z"/><path d="M483.105 224.949c-9.717-8.094-13.741-11.93-11.921-21.398 5.588-10.328 8.816-22.464 8.816-35.441 0-21.712-9.036-41.065-23.058-53.523-8.67-14.834-24.283-25.463-43.124-29.246-20.05-4.027-40.644.458-55.09 11.994C321.682 126.918 290 140.109 256 140.109s-65.682-13.192-102.728-42.776c-14.446-11.535-35.04-16.02-55.09-11.994-18.841 3.783-34.454 14.412-43.124 29.246C41.036 127.044 32 146.397 32 168.109c0 12.977 3.228 25.113 8.816 35.441 1.82 9.468-2.204 13.305-11.921 21.398C16.701 235.106 0 249.018 0 284.109c0 30.939 44.4 68.017 85.951 92.4 55.883 32.792 117.864 51.6 170.049 51.6s114.166-18.808 170.049-51.6C467.6 352.126 512 315.049 512 284.109c0-35.091-16.701-49.003-28.895-59.16zM464 168.109c0 10.585-2.696 20.437-7.312 28.661-.078.131-.148.266-.219.402-7.916 13.829-21.31 22.937-36.47 22.937-24.262 0-44-23.327-44-52s19.738-52 44-52c9.282 0 17.896 3.423 25.002 9.245.351.356.731.674 1.135.955 10.829 9.481 17.864 24.686 17.864 41.8zM65.857 126.314a7.967 7.967 0 001.144-.962c7.105-5.82 15.718-9.242 24.999-9.242 24.262 0 44 23.327 44 52s-19.738 52-44 52c-15.159 0-28.552-9.107-36.468-22.935-.072-.138-.143-.275-.222-.407-4.614-8.223-7.31-18.074-7.31-28.658 0-17.111 7.032-32.315 17.857-41.796zm352.094 236.395c-52.713 30.933-113.256 49.4-161.951 49.4s-109.238-18.467-161.951-49.4C48.095 335.744 16 303.422 16 284.109c0-27.596 11.761-37.393 23.135-46.867 5.547-4.621 11.494-9.583 15.006-16.426 10.334 9.552 23.514 15.293 37.858 15.293 33.084 0 60-30.505 60-68 0-33.803-21.879-61.914-50.429-67.125 15.396-3.025 30.975.273 41.717 8.852 40.075 32.002 74.837 46.273 112.712 46.273s72.637-14.271 112.712-46.273c10.743-8.579 26.321-11.878 41.717-8.852C381.879 106.195 360 134.307 360 168.109c0 37.495 26.916 68 60 68 14.344 0 27.524-5.741 37.858-15.293 3.512 6.842 9.459 11.805 15.006 16.426C484.239 246.717 496 256.513 496 284.109c0 19.313-32.095 51.635-78.049 78.6z"/><path d="M92 180.109c8.445 0 36-1.154 36-16s-27.555-16-36-16-36 1.154-36 16 27.555 16 36 16zM384 164.109c0 14.846 27.555 16 36 16s36-1.154 36-16-27.555-16-36-16-36 1.154-36 16zM232 228.109a8 8 0 008-8v-8a8 8 0 00-16 0v8a8 8 0 008 8zM280 228.109a8 8 0 008-8v-8a8 8 0 00-16 0v8a8 8 0 008 8z"/></svg>
```

**Review by rossbulat — COMMENTED** [review]
Thank you! Just one small revision on the SVG.

**Review by frogstaking — COMMENTED** [review]
Yes of course

File: library/validator-assets/src/Frog.tsx (hunk 1 above)
Comment (frogstaking) [reply library/validator-assets/src/Frog.tsx]: Just updated

**Pull request merged**